packages/askui-nodejs/src/execution/ui-control-client.ts

```diff
@@ -721,136 +677,51 @@ export class UiControlClient extends ApiCommands {
    * const nonExistentElements = exists.elements.filter((e) => e.exists===false)
    * ```
    *
-   * @param {ExpectExistenceInputParameter[]} params - Objects containing the required properties
-   *                                                   'type', 'label', 'matching' and 'relation'.
-   * @property {string} params.type - The type of the element: 'text' | 'button' | 'checkbox' |
-   *                                                           'switch' | 'textfield' | 'element'
-   * @property {string} params.label - The label text for the element.
-   *                                   In case of 'text': The text to be matched for
-   * @property {string} params.matching - Whether the text is matched using similarity,
-   *                                      exact match or a regular expression.
+   * @param {ExpectExistenceInputParameter[]} params - Objects containing the required property
+   *                                                   'type' and the optional properties
+   *                                                   'text' and 'relation'.
+   * @property {string} params.type - The type of the element: 'otherElement' | 'switch' |
+   *                                  'element' | 'container' | 'checkbox' | 'element' |
+   *                                  'button' | 'table' | 'text' | 'icon' | 'image' | 'textfield'
+   * @property {Object} params.text - Object containing value and matching strategy.
```

Comment:
Why not be explicit about this and the other (optional) params / params with default values (see https://jsdoc.app/tags-param#optional-parameters-and-default-values)? 🙂